UPGRADING.md

```diff
@@ -3,7 +3,13 @@ Upgrading Grape
 
 ### Upgrading to >= 3.0.0
 
-### Endpoint execution simplified and `return` deprecated
+#### Configuration API Migration from ActiveSupport::Configurable to Dry::Configurable
+
+Grape has migrated from `ActiveSupport::Configurable` to `Dry::Configurable` for its configuration system since its [deprecated](https://github.com/rails/rails/blob/1cdd190a25e483b65f1f25bbd0f13a25d696b461/activesupport/lib/active_support/configurable.rb#L3-L7)
```

Comment:
```suggestion
Grape has migrated from `ActiveSupport::Configurable` to `Dry::Configurable` for its configuration system since its [deprecated](https://github.com/rails/rails/blob/1cdd190a25e483b65f1f25bbd0f13a25d696b461/activesupport/lib/active_support/configurable.rb#L3-L7).
```